protos/icmp/icmp.go

```diff
@@ -0,0 +1,318 @@
+package icmp
+
+import (
+	"net"
+	"time"
+
+	"github.com/elastic/libbeat/common"
+	"github.com/elastic/libbeat/logp"
+	"github.com/elastic/libbeat/publisher"
+
+	"github.com/elastic/packetbeat/config"
+	"github.com/elastic/packetbeat/protos"
+
+	"github.com/tsg/gopacket/layers"
```

Comment:
Good point. I suggest we migrated all "references" at one stage. So linking to tsg is fine here.